src/app/modules/alerts/components/alerts-panel/alerts-panel.component.spec.ts

```diff
@@ -93,8 +104,12 @@ describe('AlertsPanelComponent', () => {
     expect(websocket.call).toHaveBeenCalledWith('alert.list');
   });
 
-  it('checks for HA status and passes it to the ix-alert', () => {
-    expect(spectator.inject(WebSocketService).call).toHaveBeenCalledWith('failover.licensed');
+  it('selects HA status from store and passes it to the ix-alert', () => {
+    spectator.inject(Store).dispatch(loadFailoverLicensedStatus());
+
+    spectator.inject(Store).select(selectIsHaLicensed).subscribe((isHaLicensed) => {
```

Comment:
This tests selector, not the component.
This line and dispatch above are not necessary.
You can also remove `failover.licensed` mock above.